com.fightcade.Fightcade.Steamos.yml

```diff
@@ -0,0 +1,61 @@
+id: com.fightcade.Fightcade.Steamos
+runtime: com.fightcade.Fightcade
+runtime-version: stable
+sdk: org.freedesktop.Sdk//24.08
+build-extension: true
+separate-locales: false
+
+modules:
+  - name: libsecret
+    buildsystem: meson
+    build-options:
+      prefix: /app/steamos
+      lib: lib
+    config-opts:
+      - -Dmanpage=false
+      - -Dvapi=false
+      - -Dgtk_doc=false
+      - -Dintrospection=false
+    cleanup:
+      - "/bin"
+      - "/include"
+      - "/lib/pkgconfig"
+      - "/share/gir-1.0"
+      - "/share/man"
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/libsecret/0.21/libsecret-0.21.6.tar.xz
+        sha256: 747b8c175be108c880d3adfb9c3537ea66e520e4ad2dccf5dce58003aeeca090
+  - name: min
+    buildsystem: simple
+    build-commands:
+      # Install the run script for the Min browser
+      - install -Dm755 run.sh ${FLATPAK_DEST}/bin/min-browser
+      - install -Dm755 apply_extra ${FLATPAK_DEST}/bin/apply_extra
+    post-install:
+      # Install the metainfo file for the Min browser
+      - >-
+        install -Dm644
+        -t ${FLATPAK_DEST}/share/metainfo/
+        ${FLATPAK_ID}.metainfo.xml
+    sources:
+      - type: extra-data
+        filename: min.deb
+        only-arches:
+          - x86_64
+        url: https://github.com/minbrowser/min/releases/download/v1.35.0/min-1.35.0-amd64.deb
+        sha256: 6d066842e8d0f0fa9aad0bd7cb5f56a32a67937ac75e8710c3b229cbcfc89220
+        size: 79754278
+      - type: file
+        path: com.fightcade.Fightcade.Steamos.metainfo.xml
+      - type: script
+        dest-filename: run.sh
+        commands:
+          - zypak-wrapper /app/steamos/extra/min/min --no-gpu-sandbox --no-sandbox "$@" 2>&1
```

Comment:
drop no-sandbox if you use zypak.

Reply:
[4f1bfb6](https://github.com/flathub/flathub/pull/6667/commits/4f1bfb64b314910e9220f1de60b4b740c05e6649)